tests/go.mod

```diff
@@ -11,7 +11,7 @@ replace (
 require (
 	github.com/pulumi/pulumi-kubernetes/provider/v2 v2.0.0-00010101000000-000000000000
 	github.com/pulumi/pulumi-kubernetes/sdk/v2 v2.0.0
-	github.com/pulumi/pulumi/pkg/v2 v2.2.2-0.20200514204320-e677c7d6dca3
-	github.com/pulumi/pulumi/sdk/v2 v2.2.2-0.20200514204320-e677c7d6dca3
+	github.com/pulumi/pulumi/pkg/v2 v2.2.2-0.20200519094106-adfa8116fbf6
+	github.com/pulumi/pulumi/sdk/v2 v2.2.2-0.20200519094106-adfa8116fbf6
```

Comment:
Minor - but can we move to a stable version of `pulumi/pulumi` now?

Reply:
Yes, I will rebase, depend on stable, and resolve conflicts tomorrow